### PR TITLE
update the output of kubectl get pods in examples/iscsi

### PR DESCRIPTION
**Pull request by caesarxuchao**
#10168

@JanetKuo @krousey @nikhiljindal @bgrant0607

### CODE REVIEW

**Comment by nikhiljindal** [comment]
LGTM, thanks!

File: examples/iscsi/README.md [hunk 1]
```diff
@@ -38,10 +38,8 @@ Here is my command and output:
 ```console
 # kubectl create -f examples/iscsi/iscsi.json
 # kubectl get pods
-POD       IP            CONTAINER(S)   IMAGE(S)           HOST                                    LABELS    STATUS    CREATED         MESSAGE
-iscsipd   10.244.3.14                                     kubernetes-minion-bz1p/104.154.61.231   <none>    Running   About an hour   
-                        iscsipd-rw     kubernetes/pause                                                     Running   About an hour   
-                        iscsipd-ro     kubernetes/pause                                                     Running   About an hour   
+NAME      READY     STATUS    RESTARTS   AGE
+iscsipd   2/2       RUNNING              2m
```
Comment (yujuhong) [inline examples/iscsi/README.md]: Shouldn't kubectl display "0" RESTARTS in this case?

Comment (caesarxuchao) [inline examples/iscsi/README.md]: Nice catch! I manually wrote this line and missed this.

Comment (caesarxuchao) [inline examples/iscsi/README.md]: Updated.

**Comment by yujuhong** [comment]
Both shippable and jenkins failures are irrelevant to this PR...

**Pull request merged**